pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -428,6 +430,12 @@ func DisableMutators(names ...string) CustomOption {
 	}
 }
 
+// minSupportedSkipVersionUpgrade is the minimum version after which
+// "skip version" upgrades are supported (i.e., upgrading two major
+// releases in a single upgrade). Note that support is currently
+// experimental and requires an environment variable.
```

Comment:
Nit: might as well mention the name of that environment variable.

Should we add a `TODO` here for the note? Presumably, it won't remain experimental indefinitely.

Reply:
If we are no longer testing 23.2 skipping, do we need the environment variable anymore? Or is it still "experimental" in 24.1 -> 24.3 (do we still need the env var)?

Reply:
We can get rid of the env var logic.